pkg/agent/run.go

```diff
@@ -170,7 +175,18 @@ func gatherAndOutputData(ctx context.Context, config Config, preflightClient *cl
 		}
 		log.Println("Data saved locally to", OutputPath)
 	} else {
-		postData(config, preflightClient, readings)
+		//TODO Change backoff parameters to those desired
```

Comment:
What do you mean with this TODO?

Reply:
The *exponentialBackoff takes certain parameters such as initial interval, and max elapsed time. We can make them all configurable (I think the config file would be the best way of doing that). But I think its best we agree on the optimal default values for the agent

Reply:
I would suggest initial interval of 5s, with max elapsed time 5m.